### ISSUE TITLE
Text translation from audio not complete

### ISSUE THREAD

**Issue by Florian9M** [ggerganov/whisper.cpp#1687]
Hello everyone!

I'm trying to extract text from an audio file. When I process a file about 20 seconds long, I only get a portion of the text contained in the audio, but not the entire thing. With a 30-second audio file, I notice that I get a bit more text compared to the 20-second one. Logically, the longer the audio duration, the more text I get. However, I never seem to get the complete text of the audio.

Has anyone else encountered this problem?

**Comment by bobqianic** [comment]
Could you share the audio sample along with the details of the model and parameters you used for transcription?

**Comment by Florian9M** [comment]
I use the model: ggml-small.bin
Regarding the 'whisper_full_params' parameters, here they are:
wparams.max_tokens = 700000;
wparams.max_len = 700000;

I am attaching the audio file in mp3.
I converted it with ffmpeg to pcm_s16le with the command below:

ffmpeg -i LE_listening_B1_A_weather_forecast.mp3 -acodec pcm_s16le -ac 1 -ar 16000 LE_listening_B1_A_weather_forecast.wav

I only kept 20 seconds of this audio with the command:
ffmpeg -ss 00:00:06 -to 00:00:26 -i LE_listening_B1_A_weather_forecast.wav -c copy LE_listening_B1_A_weather_forecast-20s-part.wav

Here is the transcript of the 20-second video: 
Hello and good morning! Well, we're off to a good start in the south this week, as most of the rain from the weekend has disappeared – just a few patches of cloud and maybe some showers here on the east coast. They'll all clear up by lunchtime, though. Over the next day or so, London

This is what whisper returns to me:
whisper_print_timings: load time = 1777.86 ms
whisper_print_timings: fallbacks = 0 p / 0 h
whisper_print_timings: mel time = 113.61 ms
whisper_print_timings: sample time = 47.29 ms / 17 runs (2.78 ms per run)
whisper_print_timings: encode time = 52428.16 ms / 1 runs (52428.16 ms per run)
whisper_print_timings: decode time = 853.76 ms / 16 runs (53.36 ms per run)
whisper_print_timings: prompt time = 133.99 ms / 1 runs (133.99 ms per run)
whisper_print_timings: total time = 53584.38 ms
Got: 1 segments
T0 of segment: 0
T1 of segment: 500
Got this segment: Hello and good morning. Well, we're off to a good start.

The result: Hello and good morning. Well, we're off to a good start.

With a 30 second audio I get: Hello and good morning! Well, we're off to a good start in the south this week, as most
